service/modification/Refund.php

```diff
@@ -96,8 +96,8 @@ public function request(OrderSlip $orderSlip, $currency)
 
         $fullRefundAmount = $orderSlip->amount;
 
-        // In case shipping cost amount is not empty add shipping costs to the order slip amount
-        if (!empty($orderSlip->shipping_cost)) {
+        // If admin wants to include shipping, add shipping costs to the order slip amount
+        if ($orderSlip->shipping_cost !== '0') {
```

Comment:
I would check for the positive case here instead of anything else than string 0, jsut to be sure to add the shipping cost when it is "enabled" = 1

```suggestion
        if ($orderSlip->shipping_cost === '1') {
```